src/commonMain/kotlin/me/alllex/parsus/tree/SyntaxTree.kt

```diff
@@ -1,5 +1,6 @@
 package me.alllex.parsus.tree
 
+import kotlin.jvm.JvmName
```

Comment:
This maybe an unused import?

Reply:
Interestingly it is not optional, it's required.